packages/devextreme-scss/scss/widgets/fluent/chat/_colors.scss

```diff
@@ -4,27 +4,107 @@
 
 // adduse
 
+/**
+* $name 10. Background color
+* $type color
+*/
 $chat-background-color: $base-bg !default;
+
+/**
+* $name 20. Border color
+* $type color
+*/
 $chat-border-color: $base-border-color !default;
-$chat-messagebox-border-color: $base-border-color !default;
+$chat-messagebox-border-color: $chat-border-color !default;
+
+/**
+* $name 30. Avatar color
+* $type color
+*/
 $chat-avatar-color: $base-text-color !default;
+
+/**
+* $name 31. Avatar background color
+* $type color
+*/
 $chat-avatar-background-color: $base-active-bg !default;
+
+/**
+* $name 40. Bubble color
+* $type color
+*/
 $chat-bubble-color-primary: $base-text-color !default;
+
+/**
+* $name 41. Bubble background color primary
+* $type color
+*/
 $chat-bubble-background-color-primary: null !default;
+
+/**
+* $name 42. Bubble background color secondary
+* $type color
+*/
 $chat-bubble-background-color-secondary: $base-border-color !default;
+
+/**
+* $name 50. Information color
+* $type color
+*/
 $chat-information-color: null !default;
 
+/**
+* $name 60. Day header color
+* $type color
+*/
 $chat-messagelist-day-header-color: null !default;
+
+/**
+* $name 61. Empty list icon color
+* $type color
+*/
 $chat-messagelist-empty-icon-color: null !default;
+
+/**
+* $name 62. Empty list icon background color
+* $type color
+*/
 $chat-messagelist-empty-icon-background-color: null !default;
+
+/**
+* $name 63. Empty list message color
+* $type color
+*/
 $chat-messagelist-empty-message-color: $base-text-color !default;
+
+/**
+* $name 64. Empty list prompt color
+* $type color
+*/
 $chat-messagelist-empty-prompt-color: null !default;
 
+/**
+* $name 70. Error color
+* $type color
+*/
 $chat-alertlist-color: $base-invalid-color !default;
 
-$chat-typingindicator-color: null !default;
+/**
+* $name 81. Typing indicator background color
+* $type color
+*/
 $chat-typingindicator-bubble-bg-color: $chat-bubble-background-color-secondary !default;
+
+/**
+* $name 82. Typing indicator circle background color
+* $type color
+*/
 $chat-typingindicator-circle-bg-color: null !default;
+
+/**
+* $name 83. Typing indicator center circle background color
```

Comment:
```suggestion
* $name 83. Typing indicator central circle color
```

Reply:
Can we also move this one above "outer circles"?

Reply:
928565d1f0814077eb111893035b73aa0945b67b